use-cases/computer_vision/1-metastases-detection-train-model.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p>This cell returned the following error: </p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
UnsupportedOperation                      Traceback (most recent call last)
&lt;ipython-input-12-0c62294fc66b&gt; in &lt;module&gt;
      2     zf.extractall()
      3     with zf.open('data/camelyon16_tiles.h5') as hf:
----&gt; 4         f = h5py.File(hf,'r')
      5 
      6         X = f['x'][()]

~/anaconda3/envs/python3/lib/python3.6/site-packages/h5py/_hl/files.py in __init__(self, name, mode, driver, libver, userblock_size, swmr, rdcc_nslots, rdcc_nbytes, rdcc_w0, track_order, fs_strategy, fs_persist, fs_threshold, **kwds)
    425                                fapl, fcpl=make_fcpl(track_order=track_order, fs_strategy=fs_strategy,
    426                                fs_persist=fs_persist, fs_threshold=fs_threshold),
--&gt; 427                                swmr=swmr)
    428 
    429             if isinstance(libver, tuple):

~/anaconda3/envs/python3/lib/python3.6/site-packages/h5py/_hl/files.py in make_fid(name, mode, userblock_size, fapl, fcpl, swmr)
    188         if swmr and swmr_support:
    189             flags |= h5f.ACC_SWMR_READ
--&gt; 190         fid = h5f.open(name, flags, fapl=fapl)
    191     elif mode == 'r+':
    192         fid = h5f.open(name, h5f.ACC_RDWR, fapl=fapl)

h5py/_objects.pyx in h5py._objects.with_phil.wrapper()

h5py/_objects.pyx in h5py._objects.with_phil.wrapper()

h5py/h5f.pyx in h5py.h5f.open()

h5py/h5fd.pyx in h5py.h5fd.H5FD_fileobj_get_eof()

UnsupportedOperation: seek
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
What version of python are you using? This error should not occur with python 3.7 https://github.com/fastavro/fastavro/issues/217

Reply:
I used a fresh notebook instance and the `conda_python3` kernel.

Reply:
Or we should make a clear note that this notebook is designed to work with which specific kernel on which environment between Studio or NI.

Reply:
Got it. I fixed it so it works with 3.6.x as well. Ali has the code and will submit a new pull request